spec/models/compliance_spec.rb

```diff
@@ -140,6 +140,36 @@
         subject { template }
         include_examples ".check_compliance"
       end
+
+      context "no condition" do
+        let(:policy) do
+          FactoryGirl.create(:miq_policy,
+                             :mode       => 'compliance',
+                             :towhat     => 'Vm',
+                             :expression => MiqExpression.new("=" => {"field" => "Vm-retired", "value" => "true"}),
+                             :active     => true)
+        end
+        let(:policy_set) { FactoryGirl.create(:miq_policy_set) }
+        let(:event_definition) { MiqEventDefinition.find_by_name("vm_compliance_check") }
+
+        before do
+          policy.sync_events([FactoryGirl.create(:miq_event_definition, :name => "vm_compliance_check")])
+          policy_set.add_member(policy)
+          ems_vmware.add_policy(policy_set)
+        end
+        subject { vm1 }
```

Comment:
Minor, but I think this gets a little lost down here and would make more sense by the let definitions.